docs/content/en/docs/operator-manual/piped/installation/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Installation"
+linkTitle: "Installation"
+weight: 1
+description: >
+  This page describes how to install a Piped.
+---
+
+Since Piped is a stateless agent, no database or storage is required to run. In addition, a Piped can interact with one or multiple cloud providers, so the number of Piped and where they should run is entirely up to your preference. For example, you can run your Pipeds in a Kubernetes cluster to deploy not just Kubernetes applications but your Terraform and ClourRun applications as well.
```

Comment:
typo
```suggestion
Since Piped is a stateless agent, no database or storage is required to run. In addition, a Piped can interact with one or multiple cloud providers, so the number of Piped and where they should run is entirely up to your preference. For example, you can run your Pipeds in a Kubernetes cluster to deploy not just Kubernetes applications but your Terraform and CloudRun applications as well.
```